pkgs/applications/misc/kjv/default.nix

```diff
@@ -1,43 +1,41 @@
-{ lib, stdenv, fetchFromGitHub, fetchpatch }:
+{ lib, stdenv, fetchFromGitHub, fetchpatch, readline }:
 
 let
+  patchPrefix = "https://github.com/samuelgrf/kjv/commit/";
 
-patch-base = "https://github.com/LukeSmithxyz/kjv/commit/";
-
-add-apocrypha = fetchpatch {
-  url = patch-base + "b92b7622285d10464f9274f11e740bef90705bbc.patch";
-  sha256 = "0n4sj8p9m10fcair4msc129jxkkx5whqzhjbr5k4lfgp6nb1zk8k";
-};
-
-add-install-target = fetchpatch {
-  url = patch-base + "f4ad73539eb73f1890f4b791d8d38dd95900a4a4.patch";
-  sha256 = "1yzj72i5fkzn2i4wl09q6jx7nwn2h4jwm49fc23nxfwchzar9m1q";
-};
+  add-apocrypha = fetchpatch {
+    url = patchPrefix + "3f5cc5cbb7b5a94d5dd8b17a450fac4a3e162a32.patch";
+    sha256 = "sha256-5xxeem3Fg1C0L+zp7Ks/T+3dSXLjpXcM5morLFQYk74=";
```

Comment:
I guess this works, but should it be `hash = ..`?

Reply:
`fetchpatch` doesn't support SRI hashes yet, therefore using the `hash` argument here results in an error, see <https://github.com/NixOS/nixpkgs/pull/79987>.

I changed this to the supported regular base32 hashes instead in the last force push.